.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
```

Comment:
```suggestion
indent_size = 2
end_of_line = lf
charset = utf-8
```

@iovis  feels like you've got some interest in watching over file formatting :)

I'd like to know your opinion about adding the missing `end_of_line` directive that defines line break representation

![Screen Shot 2022-10-14 at 16 41 26](https://user-images.githubusercontent.com/983242/195874764-23d3581c-9340-498c-9828-ca5e43e8f4f9.jpg)

It's not present either on the private repo, so I'm wondering if it was omitted on purpose or maybe the file was created way before that option was available.

Reply:
`lf` is fine, thank you!